umbraco/Umbraco-CMS
# Makes sure the nucache is not truncated when its null

### Prerequisites


### Description

This pull request should fix #14253  by changing the checks in the `NuCacheContentRepository.Rebuild` to make sure that it does not truncate the table if null is passed as parameters

### Test
    1. call: NuCacheContentRepository.Rebuild(null, null, null)
    2. see the table cmsContentNu is not empty after the call

https://github.com/umbraco/Umbraco-CMS/issues/14253
